components/Flipper/FlipCustomize/FlipCustomize.tsx

```diff
@@ -516,6 +516,21 @@ function FlipCustomize() {
                                     type="checkbox"
                                 />
                             </Form.Group>
+                            <Form.Group>
+                                <Form.Label className={styles.label} htmlFor="modCountdown">
+                                    Set maxCost to purse
```

Comment:
```suggestion
                                    Don't show flips costing more than purse
```
The behavior will probably change to the purse not modifying maxCost because many settings updates cause issues

Reply:
I changed it to `Don't show flips that <br /> cost more than purse` because the width of the text doesnt fit well into the limited space of the labels